src/ImageSettings.hs

```diff
@@ -0,0 +1,13 @@
+module ImageSettings
+  ( ImageSettings
+    , imageWidth
+    , imageHeight
+    , fieldOfView
+  ) where
+
+
+data ImageSettings = ImageSettings
+ { imageWidth :: Int
+ , imageHeight :: Int
+ , fieldOfView :: Double
```

Comment:
I'd prepend those with `get` (`getImageWidth`, `getImageHeight`, `getFieldOfView`)

Reply:
Sure, it makes sense, since they're really just getters.